application/packages/python/ftrs_data_layer/models.py

```diff
@@ -170,6 +171,71 @@ class Telecom(BaseModel):
     web: str | None
 
 
+class OpeningTimeCategory(str, Enum):
```

Comment:
```suggestion
class OpeningTimeCategory(str, Enum):
    AVAILABLE_TIME = "availableTime"
    AVAILABLE_TIME_VARIATIONS = "availableTimeVariations"
    AVAILABLE_TIME_PUBLIC_HOLIDAYS = "availableTimePublicHolidays"
    NOT_AVAILABLE = "notAvailable"

```
to follow the Python's convention for constants and enum members to be UPPERCASE